src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -96,13 +96,13 @@
           + "    FILE_FORMAT = [<fileFormat>]\n"
           + ") AS SELECT "
           + "<if(isCSV)>newid() as datarepo_row_id,\n       "
-          + "<columns:{c|<c.name>}; separator=\",\n       \">"
+          + "<columns:{c|[<c.name>]}; separator=\",\n       \">"
           + "<else>"
           + "newid() as datarepo_row_id,\n       "
           + "<columns:{c|"
           + "<if(c.requiresJSONCast)>"
-          + "cast(JSON_VALUE(doc, '$.<c.name>') as <c.synapseDataType>) <c.name>"
-          + "<else>JSON_VALUE(doc, '$.<c.name>') <c.name>"
+          + "cast(JSON_VALUE(doc, '$.[<c.name>]') as <c.synapseDataType>) [<c.name>]"
+          + "<else>JSON_VALUE(doc, '$.[<c.name>]') [<c.name>]"
```

Comment:
huh...we need the brackets when pulling out of JSON?  We should make sure it's actually reading out those values as expected

Reply:
Should be fixed now.